### PR TITLE
Card styles adjustment

### PR DESCRIPTION
**Pull request by PaigeVannelli**
- Add minor card styling adjustments

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -510,47 +524,39 @@ header {
...
-    background-image: url("https://static.scientificamerican.com/blogs/cache/file/BF34DFF9-BBE0-4079-B998E97CA9EAFEB1_source.jpg?w=590&h=800&B7A98B9B-C3E5-4106-9CE11BC335728795");
-}
-
-.layer {
-    background-color: rgba(194, 194, 194, 0.7);
-    opacity: 90%;
-    top: 0;
-    left: 0;
-    height: 100%;
+    background-image: linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url("https://static.scientificamerican.com/blogs/cache/file/BF34DFF9-BBE0-4079-B998E97CA9EAFEB1_source.jpg?w=590&h=800&B7A98B9B-C3E5-4106-9CE11BC335728795");
```
Comment (jordy1611) [inline styles.css:559]: Probably a way to only do this once but you're not using any JS so I wouldn't worry too much about it

File: index.html [hunk 2]
```diff
@@ -17,33 +17,31 @@
...
         <article class="card" class="card1">
             <section class="card-top card-top1">
-              <div class="layer">
-                <div class="card-buttons">
-                  <button class="arrow-button"><img class="arrow" src="./assets/redo-arrow.svg"></button>
-                  <div class="card-number-container">
-                    <p class="card-label">views</p>
-                    <p class="card-number">01</p>
-                  </div>
+              <div class="card-buttons">
```
Comment (jordy1611) [inline index.html:29]: Avoid divs unless they're just for styling. Don't do anything when it comes to accessibility

**Review by jordy1611 — APPROVED** [review]
Looks good! Only thing I can think of is using background images instead of img's in html means you can't use an alt tag. Not sure what the equivalent there is but I'd keep that in mind when you're making accessibly a priority.

File: styles.css (hunk 1 above)
Comment (PaigeVannelli) [reply styles.css:559]: Yeah good call out. If I were to do it over I wouldn't make them background images I think

**Pull request merged**